android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -486,12 +486,13 @@ class MusicService : HeadlessJsTaskService() {
             event.notificationStateChange.collect {
                 when (it) {
                     is NotificationState.POSTED -> {
-                        if (AppForegroundTracker.foregrounded && it.notificationId == MUSIC_NOTIFICATION_ID) {
-                            startForeground(it.notificationId, it.notification)
-                        }
+                        startForeground(it.notificationId, it.notification)
```

Comment:
This line causing a crash as I mentioned [here](https://github.com/doublesymmetry/react-native-track-player/pull/1974#issuecomment-1487575919), @puckey wdyt about this quick fix 
```
if (Build.VERSION.SDK_INT < Build.VERSION_CODES.S) {
                            startForeground(it.notificationId, it.notification)
    }
```

Reply:
@MortadhaFadhlaoui Could you open a new pr for this with some explanation on your changes? Thanks!